src/sentry/conf/server.py

```diff
@@ -920,6 +938,13 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     },
 }
 
+# We prefer using crontab, as the time for timedelta will reset on each deployment. More information:  https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html#periodic-tasks
+TIMEDELTA_ALLOW_LIST = {
+    "flush-buffers",
+    "sync-options",
+    "schedule-digests",
+}
```

Comment:
Why are these specific tasks whitelisted?

Reply:
They're setup with a specific number of seconds, which I wasn't able to get working with `crontab`